pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -17,7 +17,7 @@ import (
 	"text/template"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
-	. "github.com/code-ready/crc/pkg/os"
+	crcos "github.com/code-ready/crc/pkg/os"
```

Comment:
we should never use `.` otherwise it is hard to trace back the functions which is consumed in the file.

Reply:
Ahh.. this was because i wan't really sure what to call our os pkg... `crcos` sounds fun!